### PR TITLE
Database schema diagram

### PR DESCRIPTION
**Pull request by codewanderlust**
> In this pull request, Ayana and I used the database schema diagram to come up with the tables
From the schema diagram, we were able to come up with the following tables `patients`, `invoices`, `treatments`, `medical_histories`, and `invoice_items`
We also designed the join table, `medical_histories_treatments` to reflect the many-to-many relationships between the `medical_histories` table and the `treatments` table

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe is professionally documented, and adhere(s) to the following standard(s):
- [ ] All main headers from the [ReadMe template](https://github.com/microverseinc/readme-template) should be present in your ReadMe, only the _FAQ_ section is optional.
- [ ]  The _Future Features_ section of your ReadMe includes 1-3 potential features to be added later. 
- [ ] Kindly ensure to remove the Microverse template logo. Optionally, you can replace it with your own logo if you have one.
- [ ] Kindly ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe. Similarly, kindly also remove guide notes for _license_ creation under the _license_ section.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+);
+
+CREATE TABLE medical_histories_treatments (
+  medical_history_id INT NOT NULL,
+  treatment_id INT NOT NULL,
+  PRIMARY KEY (medical_history_id, treatment_id),
+  FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id),
+  FOREIGN KEY (treatment_id) REFERENCES treatments(id)
+);
+
```
Comment (ShadyShawkat) [inline schema_based_on_diagram.sql:47]: - [ ] Kindly make sure to create indexes for each of the created foreign keys, as per [this](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md#:~:text=Remember%20to%20add%20the%20FK%20indexes.) requirement.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (rloterh) [reply schema_based_on_diagram.sql:47]: resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Roy & Alelign,  

Your project is complete! 🏁  There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Good job implementing recommended changes!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

**You can also consider:**
- Kindly include details of the other co-contributor(s) in your ReadMe.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers..Happy coding!  💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**